tests/test_crash_similarity.py

```diff
@@ -62,12 +61,11 @@ def test_order_similarity(self):
 
     def test_clean_func(self):
```

Comment:
They will be moved to `test_utils.py` at some point, right? So the best time to move the tests is also when you move the functions.
This allows us to have a clean history of what is happening in the repository.